### PR TITLE
:sparkles: pagination for frequentlymutatedgenes

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: app/react-components/components/FrequentlyMutatedGenesContainer.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+export default
+compose(
+  withState('state', 'setState', { data: { hits: [], total: 0 }, loading: true }),
+  withProps({
+    fetchData: async props => {
+      props.setState(s => ({ ...s, loading: true }));
+
+      const url =
+        `${props.config.es_host}/${props.config.es_index_version}-gene-centric/gene-centric/_search`;
```
Comment (Jephuff) [inline app/react-components/components/FrequentlyMutatedGenesContainer.js]: we shouldn't be calling `es_host` any more, would we switch this to hit the api?

Comment (alex-wilmer) [inline app/react-components/components/FrequentlyMutatedGenesContainer.js]: right thanks, will swap

**Comment by alex-wilmer** [comment]
@Jephuff fixed. Though I think maybe `offset` isn't being handled on this endpoint for some reason? `/analysis/top_mutated_genes_by_project` is that possible?

**Comment by Jephuff** [comment]
Yep, you are correct, it is ignoring it. Do we want to change the name to `offset` or leave it as `from` (it's not handling either right now)

**Comment by Jephuff** [comment]
https://github.com/NCI-GDC/gdcapi/tree/feat/analysis-handle-from this branch accepts `from` I can change it to `offset` if we prefer that.

**Comment by shanewilson** [comment]
use whatever the other endpoints use

**Comment by alex-wilmer** [comment]
offset seems to work here on this endpoint `analysis/frequent_mutations_by_project`

**Comment by shanewilson** [comment]
all the main api/ end points use `from` - the analysis endpoints should follow that

**Comment by Jephuff** [comment]
👍 I'll update

**Pull request merged**